apiserver/facade/interface.go

```diff
@@ -213,10 +213,10 @@ type ModelMigrationFactory interface {
 	ModelImporter() ModelImporter
 }
 
-// ServiceFactory defines an interface for accessing all the services.
-type ServiceFactory interface {
-	// ServiceFactory returns the services factory for the current model.
-	ServiceFactory() servicefactory.ServiceFactory
+// DomainServices defines an interface for accessing all the services.
+type DomainServices interface {
+	// DomainServices returns the services factory for the current model.
```

Comment:
Do we still want to call this a "services factory" in comments?

Reply:
Probably not.